Lib/sysconfig.py

```diff
@@ -131,6 +131,13 @@ def joinuser(*args):
 _USER_BASE = None
 
 
+# NOTE: site.py has copy of this function.
+# Sync it when modify this function.
+def _get_implementation():
+    if '__pypy__' in sys.builtin_module_names:
+        return 'PyPy'
+    return 'Python'
```

Comment:
Would it be possible to ``import site`` here and use the ``site._get_implementation()``?

Reply:
The functionality is only in `site.py` for a performance tweak. 
> sysconfig and it's dependencies are relatively large but site module needs very limited part of them. To speedup startup time, we have copy of them.

Maybe we should remove the performance tweak instead? `site.py` already imports os, so I wonder if the statement is still true.